platform/ios/PolyPodApp/PolyPodTests/OIDAuthTests.swift

```diff
@@ -0,0 +1,282 @@
+import AppAuth
+@testable import PolyPod
+import Security
+import XCTest
+
+final class OIDAuthTests: XCTestCase {
+    private lazy var redirectURL = URL(string: "coop.polypod.com://redirect")!
```

Comment:
OK, same format here. Weird, but I guess that's OK. It might help if there were an auxiliary function that created these URLs (with some name that would indicate why this is so)

Reply:
This is the usual format used in mobile apps for deep linking.

Reply:
As stated, I guess that's OK. Still, reading the scheme from somewhere and/or create a function that would be called `deepLinkForRoute` might help understand what the literal is about...